Firmware/code

```diff
@@ -0,0 +1,21 @@
+import dearpygui.dearpygui as dpg
+dpg.create_context()
+
+USB_list=[1,2,3,4,5,6,7,9]
+
+def printValue(Sender):
+    print(dpg.get_value(Sender))
+
+with dpg.window(label="Send Message to USB Port",width=600,height=400):
```

Comment:
It would be nice to have a very clear main function rather than having the main code outside of functions, kinda like this:

`def main():
    # Your main program logic goes here
    print("Hello from the main function!")

if __name__ == "__main__":
    main()`